### PR TITLE
Added All Hunting Quests

### PR DESCRIPTION
**Pull request by Fuhhue**
Adds all hunting quests and boss hunting quests.

### CODE REVIEW

File: system/scripts/quests/beginner_uladh/1000034_hunting_spiders.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	public override void Load()
+	{
+		SetId(1000034);
+		SetName("Hunt 5 White Spiders");
+		SetDescription("You can find the white spider near the graveyard easily. The graveyard is often damaged by the white spider that have increased suddenly. Can you take the lead and banish the white spider? First hunt 5 white spiders. You don't have to come back and report. When you're done, just get your payment. - Duncan -");
+		
+		SetReceive(Receive.Automatically);
+		AddPrerequisite(ReachedTotalLevel(12));
+
+
```
Comment (exectails) [inline system/scripts/quests/beginner_uladh/1000034_hunting_spiders.cs]: Try to minimize unnecessary empty lines.

File: system/scripts/quests/beginner_uladh/1000035_hunting_foxes.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+//--- Aura Script -----------------------------------------------------------
+// Hunt 5 White Spiders
```
Comment (exectails) [inline system/scripts/quests/beginner_uladh/1000035_hunting_foxes.cs]: Name

File: system/scripts/quests/beginner_uladh/1000046_hunting_skeletons.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
+//--- Aura Script -----------------------------------------------------------
+// Hunt 5 Skeletons
+//--- Description -----------------------------------------------------------
+// Thirteenth hunting quest beginner quest series, started automatically
+// after completing Hunt 5 Skeleton Wolves.
```
Comment (exectails) [inline system/scripts/quests/beginner_uladh/1000046_hunting_skeletons.cs]: Wolves? Maybe read through all the comments and descriptions and stuff one more time.

File: system/scripts/quests/beginner_uladh/1000055_hunting_giant_spider.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	public override void Load()
+	{
+		SetId(1000055);
+		SetName("Hunt 1 Giant Spider");
+		SetDescription("I am Trefor, the guard. Have you ever heard about the Giant Spider? Why don't you drop a Common Item on the Alby dungeon altar to enter and hunt 1 Giant Spider? I think it's a good chance to test your courage. - Trefor -");
+		
+		SetReceive(Receive.Automatically);
+		AddPrerequisite(ReachedTotalLevel(15));
+
+
```
Comment (exectails) [inline system/scripts/quests/beginner_uladh/1000055_hunting_giant_spider.cs]: And check for empty lines. Those aren't a problem obviously... it just really bugs me =<

File: system/scripts/quests/beginner_uladh/1000056_hunting_small_golem.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
+//--- Aura Script -----------------------------------------------------------
+// Hunt 1 Small Golem
+//--- Description -----------------------------------------------------------
+// Second hunting quest Boss quest series, started automatically
+// after Completing Hunt 1 Giant Spider.
```
Comment (exectails) [inline system/scripts/quests/beginner_uladh/1000056_hunting_small_golem.cs]: Not a Spider.

File: system/scripts/scripts_quests.txt [hunk 6]
```diff
@@ -10,4 +10,34 @@ quests/beginner_uladh/202006_duncans_call.cs
 quests/beginner_uladh/skills_rest.cs
 quests/beginner_uladh/school_1.cs
 
-include "/user/scripts/scripts_quests.txt"
+// Hunting
+quests/beginner_uladh/1000034_hunting_spiders.cs
```
Comment (exectails) [inline system/scripts/scripts_quests.txt]: I guess you could move them into a sub-folder "hunting" or something, less clutter.

**Comment by Fuhhue** [comment]
my bad i thought i fixed all the descriptions but i guess i missed some.

File: system/scripts/quests/beginner_uladh/1000056_hunting_small_golem.cs (hunk 5 above)
Comment (Fuhhue) [inline system/scripts/quests/beginner_uladh/1000056_hunting_small_golem.cs]: It says after completing hunt 1 giant spider <- how you get the quest

Comment (exectails) [inline system/scripts/quests/beginner_uladh/1000056_hunting_small_golem.cs]: Oops, my bad^^

File: system/scripts/quests/beginner_uladh/1000046_hunting_skeletons.cs (hunk 3 above)
Comment (Fuhhue) [inline system/scripts/quests/beginner_uladh/1000046_hunting_skeletons.cs]: Same as above, you get the quest after completing Hunt 5 Skeleton Wolves.

**Comment by Fuhhue** [comment]
ill re-pull this once i get home and sync the files

**Pull request closed without merging**